traffic_ops/testing/api/v4/parameters_test.go

```diff
@@ -157,7 +176,31 @@ func UpdateTestParameters(t *testing.T) {
 	if respParameter.Value != expectedParameterValue {
 		t.Errorf("results do not match actual: %s, expected: %s", respParameter.Value, expectedParameterValue)
 	}
+}
 
+func UpdateParametersEmptyConfigFile(t *testing.T) {
+	if len(testData.Parameters) > 0 {
```

Comment:
This test will silently pass if the test data is in a bad state and the test cannot proceed

Reply:
Added failure checks